Controller/Redirect/Process.php

```diff
@@ -301,8 +301,8 @@ public function execute()
                         )
                     );
                     $this->logger->addDebug(__METHOD__ . '|5|');
+                    $this->secondChance();
                     return $this->_redirect('/');
```

Comment:
to which page is the user redirected here, after clicking on the url from the email? homepage? did we have a configurable url to be redirected to?

Reply:
yes homepage, we don`t have configurable for 791 case